### DISCUSSION TITLE
Bagaimana Cara merapihkan render text di R Shiny Dashboard

### DISCUSSION THREAD

**Discussion by takdirzd** [teamalgoritma/community#234] [Q&A] [answered]
Bagaimana cara merapihkan text tersebut dengan rapih, punya enter seperti di codenya, tapi saat di run app, textnya malah tersambung semua. Terima kasih

![image](https://user-images.githubusercontent.com/88231577/182146940-15e17837-8760-494d-8d5d-3d503cb6cd1d.png)
![image](https://user-images.githubusercontent.com/88231577/182147047-730c30bd-8357-4ac6-92f3-fd2820260786.png)

hasil yang saya inginkan seperti ini, ada spasi enternya

![image](https://user-images.githubusercontent.com/88231577/182157826-f45a507b-6af5-427b-9b19-57a74ccdcd50.png)

**Comment by amaliasf** [comment, marked as answer]
Halo Pak @takdirzd 
Untuk merapikan teks tersebut, termasuk membuat tulisan menjadi bold dan menambahkan bullets, Bapak bisa menggunakan tag `html` dan menambahkan `htmlOutput()` pada bagian UI.

Berikut referensi terkait:
- [How to create new line and bold text in R Shiny](https://stackoverflow.com/questions/39985307/shiny-r-rendertext-paste-new-line-and-bold)
- [How to add bullet points in R Shiny](https://stackoverflow.com/questions/22923784/how-to-add-bullet-points-in-r-shinys-rendertext)

Semoga membantu Pak :)

**Reply by takdirzd** [reply]
Terima kasih kak @amaliasf , sudah bisa
